.travis.yml

```diff
@@ -13,7 +13,7 @@ dist: trusty
 env:
   - YTDL_TEST_SET=core
   - YTDL_TEST_SET=download
-matrix:
+jobs:
```

Comment:
What's the point of this?

Reply:
`jobs` is the new key, `matrix` becomes the alias of it

https://github.com/travis-ci/travis-yml/blob/master/schema.json#L546